### PR TITLE
Fix auth validation for all site pages

### PR DESCRIPTION
**Pull request by Recursively**
Simple change that requires login for all pages except the home page

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,4 +2,12 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_action :auth_user!
+
+  private
+
+  def auth_user!
+    authenticate_user! unless params[:controller] == 'home'
```
Comment (TanaTanoi) [inline app/controllers/application_controller.rb]: is this worth constantizing `'home'` into `HomeController::HOME`?

Comment (Recursively) [inline app/controllers/application_controller.rb]: I wonder if :home works

Comment (Recursively) [inline app/controllers/application_controller.rb]: It does not

**Comment by Recursively** [comment]
@TanaTanoi Pushed change

**Pull request merged**